stdlib/LinearAlgebra/docs/src/index.md

```diff
@@ -516,6 +516,7 @@ Base.:^(::AbstractMatrix, ::Number)
 Base.:^(::Number, ::AbstractMatrix)
 LinearAlgebra.log(::StridedMatrix)
 LinearAlgebra.sqrt(::StridedMatrix)
+LinearAlgebra.cbrt(::AbstractMatrix{<:Real})
```

Comment:
Should this function only be for `StridedMatrix{<:Real}`, analogous to `sqrt`?

I seem to recall that restriction is there because we didn't want you accidentally calling it for a sparse matrix or similar structure where the algorithm would be insanely slow (if it worked at all).

Reply:
Thank you, @stevengj. I'll get back to you soon about this.

Reply:
Hello @stevengj. Sorry for the delay in my response. I took a detailed look at the comment. Unfortunately, a few things are still unclear. I've summarised my observations and presented my follow-up questions below.

1. The signature in the `index.md` file does not seem to influence the final signature in the help documents. E.g., for `sqrt`, although `index.md` specifies `::StridedMatrix`, the signature in the help document (HTML) is the one from the code comments: `sqrt(A::AbstractMatrix)`.
2. When I changed the signature of `cbrt` to `LinearAlgebra.cbrt(::StridedMatrix{<:Real})`, I observed no changes in the final help document.

Would you please let me know if you'd like me to still go ahead and incorporate this change? Or, did you intend that I change all signatures to `StridedMatrix`? In this regard, I'd like to mention that the `sqrt` matrices are (also) defined for `AbstractMatrix`.

Kindly let me know.

Reply:
Since `sqrt` is defined for `AbstractMatrix`, we can leave this as-is.